pkgs/development/python-modules/mariadb/default.nix

```
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, libmysqlclient, lib }:
+buildPythonPackage rec {
+  pname = "mariadb";
+  version = "1.0.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-S3sbMHowo8sSJy2bHcoNE7Zr09QVtKzrCkB/CYPzQaM=";
+    extension = "zip";
+  };
+
+  nativeBuildInputs = [
+    libmysqlclient
+  ];
+
+  meta = with lib; {
+    description = "MariaDB Connector/Python";
+    homepage = "https://github.com/mariadb-corporation/mariadb-connector-python";
+    license = licenses.lgpl21;
```

License is is not clear from my point of view. `setup.py` says [`LGPLv2+`](https://github.com/mariadb-corporation/mariadb-connector-python/blob/master/setup.py#L53) and [`LGPL 2.1`](https://github.com/mariadb-corporation/mariadb-connector-python/blob/master/setup.py#L73). LICENSE file content is `GNU LESSER GENERAL PUBLIC LICENSE Version 2.1`.

```suggestion
    license = licenses.lgpl21Only;
```

Best would be if you ask upstream for clarification. Looks like that it could be `lgpl21Only`.

---

The upstream GitHub repo issues is disabled : (

---

https://mariadb-corporation.github.io/mariadb-connector-python/license.html